website/docs/core-concepts/connecting-to-data-sources/connecting-to-databases.md

```diff
@@ -1,63 +1,95 @@
+* [Redis](/reference/datasources/querying-redis)
+* [Microsoft SQL Server](/reference/datasources/querying-mssql)
+* [Firestore](/reference/datasources/querying-firestore)
+* [Redshift](/reference/datasources/querying-redshift)
+* [Amazon S3](/reference/datasources/querying-amazon-s3)
+* [Snowflake](/reference/datasources/querying-snowflake-db)
+* [ArangoDB](/reference/datasources/querying-arango-db)
+* [SMTP](/reference/datasources/using-smtp)
 
-![](</img/create_ds.gif>)
+Appsmith can also seamlessly connect with most other databases through the RESTful API plugin. For more information, see [Integrations](/learning-and-resources/integrations)
 
-:::tip
-Databases configured within one page of an app can be accessed by developers anywhere within the workspace.
+### Sample databases
+
+
+Appsmith provides two sample datasets to help you connect the data and test the platform. 
+
+* MongoDB database with a `movies` collection
+* PostgreSQL database with a `users` table
+
+:::info Important
+The sample databases are public and shared by all users, so ensure that you don't write any confidential information during testing. As data may be added or updated by different users, the databases are automatically reset every day, so any updates made to these databases are temporary.
 :::
 
-## General Notes
+<VideoEmbed host="youtube" videoId="TrV8h_Dvhbg" title="Using A Sample Database" caption="How to use sample database"/>
+
+1. On the **Explorer** tab, click the **+** icon next to Datasources.
+2. Select the sample `users` PostgreSQL database.
+3. Go to **Datasources** and click the **Active** tab. Select the **New API +** button next to the `users` sample database.
+4. Write the query in the query editor and click the **Run** button to execute it. For example:
+
+```sql
+SELECT * FROM users;
+```
+The select query retrieves all the data from the users table. You can display the query results in a Table widget by [binding the data](/core-concepts/data-access-and-binding/displaying-data-read#displaying-data-in-a-widget) in the Table Data property.
+
+
+## General notes
+
+:::info
+Appsmith encrypts all your datasource credentials and stores them securely. Appsmith also doesn't store any data returned from your datasources and acts only as a proxy layer to orchestrate the execution of Queries. As Appsmith is an open source framework, you can [deploy it on-premise](/getting-started/setup), and audit it to ensure none of your data leaves your VPC. For more information, see [Security](/product/security#security-measures-within-appsmith). 
+:::
+
+Appsmith creates a new connection pool with the database server when you first connect the database to your application. All subsequent queries executed by Appsmith against your database then re-use this connection to ensure that your queries are executed at run-time. In the case that an idle connection is closed by the database server, Appsmith creates a new connection while executing the next query.
```

Comment:
Should the first line be `new connection pool...` or `new connection` ?

Reply:
I agree. Let's add a heading above this para called **Connection pooling** and and update the sentence as Sumit suggested,